### ISSUE TITLE
There was an error in your Gemfile, and Bundler cannot continue. 

### ISSUE THREAD

**Issue by yyl** [natew/obtvse#53]
Hi all,
I just come across this problem when I try to "bundle install" obtvse:

/Users/yyl/Desktop/NateW-obtvse-3b6beb8/Gemfile:49:in `evaluate': compile error
/Users/yyl/Desktop/NateW-obtvse-3b6beb8/Gemfile:40: syntax error, unexpected ':', expecting kEND
  gem 'sqlite3', '~> 1', platform: [:ruby, :mswin, :mingw]
                                  ^ (SyntaxError)
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/definition.rb:18:in`build'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler.rb:135:in `definition'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/cli.rb:220:in`install'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/vendor/thor/task.rb:22:in `send'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/vendor/thor/task.rb:22:in`run'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/vendor/thor/invocation.rb:118:in `invoke_task'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/vendor/thor.rb:263:in`dispatch'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/lib/bundler/vendor/thor/base.rb:386:in `start'
       /Library/Ruby/Gems/1.8/gems/bundler-1.1.3/bin/bundle:13
       /usr/bin/bundle:23:in`load'
       /usr/bin/bundle:23
There was an error in your Gemfile, and Bundler cannot continue.

I googled it but found no luck also I try to fix that file but nothing works. I am absolutely new to ruby, could anyone kindly point out where it went wrong? Thanks!

**Comment by yyl** [comment]
Oh I just find how to make it work by modifying the gem file:
line 14: gem 'stringex', '~> 1', :git=> 'git://github.com/rsl/stringex.git'
line 40: gem 'sqlite3', '~> 1', :platforms => [:ruby, :mswin, :mingw]

I am not sure if this is the correct way to do this but it does make the command run!

**Comment by Tuinslak** [comment]
Same issue on both my Mac (1.5) and Debian (Squeeze).

Modifying the Gemfile as stated solved the issue.

**Comment by joelmoss** [comment]
The problem here is that the code is designed to be run on Ruby 1.9.3, but you are running it with an earlier version of Ruby.

**Comment by ishanthukral** [comment]
I'm on Ruby 1.9.3-p194 and still having the issue. 
![Screen Shot 2012-12-10 at 9 45 34 PM](https://f.cloud.github.com/assets/3012375/5030/f4293652-433c-11e2-97d3-b99ddb6f3bbf.png)

**Comment by natew** [comment]
Ah, your bundler is out of date.  Try gem update bundler

**Comment by ishanthukral** [comment]
Thanks for your reply!

ip-XX-XX-XX-XXX:obtvse **USER**$ gem update bundler
Updating installed gems
Nothing to update

I made the suggested changes but it still gives me an error while bundle install:

Installing pg (0.13.2) with native extensions 
Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.

I tried, gem install pg -v '0.13.2', which was successful. But still the same error.
